ingestion-beam/src/main/java/com/mozilla/telemetry/posthog/PosthogEvent.java

```diff
@@ -117,7 +117,15 @@ public ObjectNode toJson() {
       properties.put("language", getLanguage());
     }
     if (getExperiments() != null) {
-      properties.set("experiments", objectMapper.valueToTree(getExperiments()));
+      for (Map.Entry<String, String> entry : getExperiments().entrySet()) {
```

Comment:
Could also be `getExperiments().foreach((key, value) -> ...)` like [here](https://github.com/mozilla/gcp-ingestion/blob/cb30cb32be2cf2488e9107aecb5baf5e65d41b3a/ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java#L337) to make this a bit shorter. But both work the same.